config.json

```diff
@@ -474,6 +474,14 @@
         "prerequisites": [],
         "difficulty": 4
       },
+      {
+        "slug": "isbn-verifier",
+        "name": "ISBN Verifier",
+        "uuid": "737bdd7e-1e69-442a-b85b-8c7580a1139e",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4
```

Comment:
4 seems a bit high. What do you think about a 3?

Reply:
I've take this from bash, however in AWK it seems more easy. 3 is ok for me